falabracman-info.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/activity/activity.info b/activity/activity.info
+index 57a2158..45d2252 100644
+--- a/activity/activity.info
++++ b/activity/activity.info
+@@ -1,10 +1,20 @@
+ [Activity]
+-name = falabracman
++name = Falabracman
+ activity_version = 2
+ host_version = 1
+-bundle_id = org.laptop.community.falabracman
++release_date = 2024-06-02
++bundle_id = org.sugarlabs.Falabracman
+ icon = activity
+ license = GPLv3+
+ exec = sugar-activity3 activity.Falabracman
++metadata_license = CC0-1.0
+ show_launcher = yes
+ repository = https://github.com/sugarlabs/falabracman-activity.git
++update_contact = tch@sugarlabs.org
++developer_name = Sugar Labs Community
++developer_id = org.sugarlabs
++tags = Education
++url = https://github.com/sugarlabs/falabracman-activity
++summary = Sugar activity to learn words
++description =  It helps in learning some words, by collecting all the letters in the right order and by avoiding the lake otherwise you will fall into it.
++screenshots = https://imgur.com/a/597YvUB https://imgur.com/a/weEN2rH https://imgur.com/a/2iuXSMr
```

Comment:
Why are this on imgur?

Reply:
No particular reason, we've been using that service for other apps under org.sugarlabs. If you think we're better served by something else, please suggest :)

Reply:
images on imgur tend to get lost, if upstream is ok with it I don't mind.

Reply:
If I am using images link which are uploaded on github than I am getting this error
`appstream-screenshots-not-mirrored-in-ostree`. May you suggest best way to include image which is better than imgur and has no problem with flatpak.

Reply:
You can ignore that. For imgur images it has to be a direct link eg. https://i.imgur.com/NS20nXf.png

Same for Github it has to be the raw.githubusercontent.com link.